integration/js/VideoEnd2EndTest.js

```diff
@@ -0,0 +1,67 @@
+const {OpenAppStep, WaitForMeetingToBeCreated, JoinVideoTestMeetingStep, ClickStartLocalVideoButton, AuthenticateUserStep, ClickStopLocalVideoButton, ClickHasStartedLocalVideoTileButton, ClickAddVideoTileButton, ClickGetLocalVideoTileButton, ClickHaveVideoTileForAttendeeIdButton, ClickSendTileStateUpdateButton, ClickGetAllVideoTilesButton, ClickGetAllRemoteVideoTilesButton, ClickHaveVideoTilesWithStreamsButton, ClickRemoveAllVideoTilesButton, WaitForRemoteUserToJoinMeeting} = require('./steps');
+const {TestUtils} = require('kite-common');
+const SdkBaseTest = require('./utils/SdkBaseTest');
+const {TileStateCheck, AddVideoTileCheck, GetAllVideoTilesCheck, HasStartedLocalVideoTileCheck, HaveVideoTilesWithStreamsCheck, GetAllRemoteVideoTilesCheck, GetLocalVideoTileCheck, HaveVideoTileForAttendeeIdCheck} = require('./checks');
+const { v4: uuidv4 } = require('uuid');
+
+class VideoTestEnd2End extends SdkBaseTest {
+  constructor(name, kiteConfig) {
+    super(name, kiteConfig, "TestAppVideoEnd2End");
+  }
+
+  async runIntegrationTest() {
+    this.numberOfParticipant = 2;
+    const attendeeId = uuidv4();
+    const meetingTitle = 'video_end2end_test_' + uuidv4() ;
+    const session = this.seleniumSessions[0];
+    const useSimulcast = this.useSimulcast;
+    await OpenAppStep.executeStep(this, session);
+    // local user join meeting step 
+    await JoinVideoTestMeetingStep.executeStep(this, session, attendeeId, meetingTitle, useSimulcast);
```

Comment:
I think we can jsut use AuthenitcateUserStep here, you dont need to pass in a meeting Title

Reply:
If I use AuthenitcateUserStep, the existing authenticate method in TestAppPage will create a new meetingTitle, which is not we want. In the existing workflow of video test, we enter the meetingTitle in the authenticate() method where we create a new meetingTitle on the fly. But what we want here is to use the meetingTitle created by the server and enter that meetingTitle in the authenticate step. In order not to break the existing test, I create this JoinVideoTestMeetingStep to pass the meetingTitle to a new method enterMeetingTitle in TestAppPage to enter the meetingTitle.